kitsune/wiki/tests/test_templates.py

```diff
@@ -1039,6 +1039,25 @@ def test_history_category_appears(self):
         eq_(200, response.status_code)
         self.assertContains(response, category[1])
 
+    def test_translation_history_with_english_slug(self):
+        """Request in en-US slug but translated locale should redirect to translation history"""
+        doc = DocumentFactory(locale=settings.WIKI_DEFAULT_LANGUAGE)
+        trans = DocumentFactory(parent=doc, locale='bn-BD', slug='bn_bd_trans_slug')
+        ApprovedRevisionFactory(document=trans)
+        # Get the page with the en-US slug
+        url = reverse('wiki.document_revisions', args=[doc.slug], locale=trans.locale)
+        response = self.client.get(url)
+        # Check redirection happens
+        eq_(302, response.status_code)
```

Comment:
You should check that the location of the response matches what you'd expect. You can do this by checking the `Location` HTTP header with syntax like `response['Location']`.